blocks/i-lens/i-lens.css

```diff
@@ -0,0 +1,274 @@
+    line-height: var(--line-height-large);
+    font-weight: var(--font-weight-bold);
+    color: var(--link-color-orange);
+}
+
+.block.i-lens .discover-more .icon {
+    font-family: icomoon, sans-serif;
+    speak: none;
+    font-style: normal;
+    font-weight: var(--font-weight-medium);
+    font-variant: normal;
+    text-transform: none;
+    line-height: 1;
+    display: inline-block;
+    vertical-align: middle;
+}
+
+.block.i-lens .discover-more .icon-up-arrow {
+    transform: rotate(90deg);
+    margin-left: var(--space-5px);
+}
+
+.block.i-lens .discover-more .icon-up-arrow::before {
+    content: "\e900";
+}
+
+.block.i-lens .i-lens-body .box {
+    background: var(--color-white);
+    padding: 1.25rem 1.5rem;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+}
+
+.block.i-lens .i-lens-body .box + .box {
+    border-top: var(--line-width-hairline) solid var(--color-lighter-gray);
+}
+
+.block.i-lens .i-lens-body .box:nth-child(even) {
+    background: var(--background-color-light-grey);
+}
+
+.block.i-lens .i-lens-body .box h3 {
+    margin-bottom: var(--space-10px);
+}
+
+.block.i-lens .i-lens-body .box ul {
+    list-style: none;
+    display: flex;
+    justify-content: normal;
+    align-items: center;
+    flex-wrap: wrap;
+    margin-left: calc(-1*var(--space-10px));
+    padding-inline-start: 0;
+}
+
+.block.i-lens .i-lens-body .box ul li {
+    flex: 0 0 33.33%;
+    max-width: 33.33%;
+    padding: var(--space-10px);
+}
+
+
+
+.block.i-lens .i-lens-body .box label {
+    font-size: var(--body-font-size-xs);
+    line-height: var(--line-height-little-small);
+    color: var(--color-light-black);
+    word-break: break-word;
+    overflow: hidden;
+    text-overflow: ellipsis;
+    display: -webkit-box;
+    height: 1.75rem;
+    -webkit-line-clamp: 2;
+    -webkit-box-orient: vertical;
+    margin-bottom: var(--space-10px);
+}
+
+.block.i-lens .i-lens-body .box .value {
+    font-size: var(--body-font-size-m);
+    line-height: var(--line-height-extra-large);
+    font-weight: var(--font-weight-semibold);
+    color: var(--color-black);
+    display: block;
+}
+
+.block.i-lens .title .icon {
+    vertical-align: middle;
+}
+
+@media (width >= 48rem) {
+    .block.i-lens .i-lens-header h3 {
+        font-size: var(--heading-font-size-ml);
+        line-height: var(--line-height-large);
+    }
+
+    .block.i-lens .discover-more {
+        text-align: right;
+    }
+
+    .block.i-lens .i-lens-body .box ul li {
+        flex: 0 0 20%;
+        max-width: 20%;
+        padding: var(--space-10px);
+    }
+
+    .block.i-lens .i-lens-body .box .value {
+        font-size: var(--body-font-size-xlm);
+    }
+
+    .block.i-lens .dropdowns .dropdown-toggle .dropdown-text {
+        white-space: nowrap;
+    }
+
+    .block.i-lens .dropdowns .dropdown-select {
+        width: auto;
+    }
+
+    .block.i-lens .dropdowns .dropdown-toggle {
+        width: auto;
+    }
+}
+
+@media (width >= 64rem) {
```

Comment:
when the numbers are big is overlapping
<img width="631" alt="image" src="https://github.com/aemsites/icicidirect/assets/46780200/6c598718-5643-43fb-8a28-cbed1c83a333">
make this breakpoint to `(width > 64rem)` have greater width then 1024 then only apply 11.11% , it fixes.
<img width="705" alt="image" src="https://github.com/aemsites/icicidirect/assets/46780200/fcc9222c-7b74-4897-92c2-289ef153c07d">

Reply:
@ravverma correct, thanks
original website has that break point at `75rem` I am going to keep that.
since at `65rem` the experience is broken.